### ISSUE TITLE
Is this library still been maintained by anyone ?

### ISSUE THREAD

**Issue by MolloKhan** [unl/rev_api#4]
Is this project abandoned ?

**Comment by mfairchild365** [comment]
Yes, this project is still maintained. Did you have any specific questions?

**Comment by MolloKhan** [comment]
Sorry for been a douche
I'm about to start developing an integration with Rev's API for a project, and I just wanted to be sure that I can rely on this library :)

**Comment by mfairchild365** [comment]
Awesome. There's are a few endpoint that we didn't need, so we didn't build them out (like uploading content). If you see anything that can be improved, pull requests are welcome.

**Comment by MolloKhan** [comment]
Sure thing bro, maybe I could contribute with the uploading content, I'll have to send my own transcripts
